Formula/openvpn.rb

```diff
@@ -63,7 +55,7 @@ def install
     (etc/"openvpn").install doc/"samples/sample-config-files/server.conf"
 
     # We don't use PolarSSL, so this file is unnecessary & somewhat confusing.
```

Comment:
Does this comment need updating too?

Reply:
:bell:

Reply:
:bellhop_bell:

Reply:
Sorry, had to sleep/study/life 😉. Yes, this comment can be renamed. Will do now.

Reply:
hehe I'm just messing with you as usual

Reply:
If I was feeling cruel I might've just left this for a few days to see whether you'd edit it yourself, but, security fixes and all that 😈.